docs/zephyr_01_high_level_requirements.sdoc

```diff
@@ -306,6 +322,21 @@ REVIEW_COMMENT: >>>
 TBD: Nothing about priorities...
 <<<
 
+[REQUIREMENT]
+UID: ZEP-122
+STATUS: Draft
+TYPE: High Level
+COMPONENT: Threads
+TITLE: Thread priority
+STATEMENT: >>>
+Threads shall have a priority.
```

Comment:
> **question:** The user should have full control over assigning priorities to threads, right? 
That is right.
And I agree this could be one of a couple RQTs for the scheduling section. But for the Thread itself I would leave it like this or do you prefer to change it?